warehouse/static/js/warehouse/index.js

```diff
@@ -119,8 +119,8 @@ docReady(() => {
 // Close modals when escape button is pressed
 docReady(() => {
   document.addEventListener("keydown", event => {
-    // Only handle the escape key press when a modal is open
-    if (document.querySelector(".modal:target") && event.keyCode === 27) {
+    if (event.keyCode === 27) {
+      window.location.href = "#modal-close";
```

Comment:
Hmm, it seems like this listener is already working as expected for other modals. I think this change will make it so that the location is changed any time the user presses the <kbd>Esc</kbd> key, which is probably not what we want.

I think it's probably not working for the hash modal because there is no "confirm" element for it to call `.cancel()` on (see below). Maybe a better solution would be to make this work in the same way as the existing modals instead?